alignak_module_backend/etc/arbiter/modules/mod-alignak_backend_arbiter.cfg

```diff
@@ -3,6 +3,9 @@ define module {
     module_types            configuration
     python_name             alignak_module_backend.arbiter
 
+    # Module log level
```

Comment:
Add in comment the values possible please

Reply:
Done